### PR TITLE
Create the Gradle project to build XSD with Gradle Plugin

### PR DESCRIPTION
**Pull request by champillon**
Create the Gradle project to build XSD with Gradle Plugin from ETDA schema for etax

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		generatePackage "${jaxbPackageName}"
+	}
+}
+
+jar {
+	archiveName "${rootProject.name}-${jarImplVersion}.jar"
+	manifest {
+		attributes (
+			'Specification-Title': "Generated ${rootProject.name} JAR",
+			'Specification-Vendor': 'Jalamor Software',
```
Comment (varokas) [inline build.gradle]: Vendor does not look correct

**Comment by varokas** [comment]
I tried to generate a schema from https://etax.rd.go.th/etax_staticpage/app/download/XMLSchemaV2.zip

but it has many issues (duplicates names, etc). 

If you fixed it, could you outline it here what changed?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+/conf/localhost.conf
+
+### Vim example.template
+[._]*.s[a-w][a-z]
+[._]s[a-w][a-z]
+*.un~
+Session.vim
+.netrwhist
+*~
+### SBT example.template
```
Comment (varokas) [inline .gitignore]: I think this .gitignore is way too long. maybe put in only what is needed

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,188 @@
+schema/
+src/main/java/th/or/etda/
```
Comment (varokas) [inline .gitignore:2]: This means we are .gitignore the entire namespace. Is this intentional?

Also normally we would generate code into "src/generated/java" for a clean separation between generated and src. but let's fix that in another PR after this is merged.

Comment (champillon) [reply .gitignore:2]: oppss!!! "src/generated/java" is the proper name, just thinking the proper name once working on this PR somehow cannot haha

File: build.gradle (hunk 1 above)
Comment (champillon) [reply build.gradle]: fixed

File: .gitignore (hunk 2 above)
Comment (champillon) [reply .gitignore]: fixed

**Comment by champillon** [comment]
On this PR, I fix the duplicate name issues and path issues on schema somehow there is some error on episode once generated, will fix in next PR

**Comment by varokas** [comment]
We are consolidating schema changes in this repo: https://github.com/ETDA/etax-schema

**Comment by varokas** [comment]
Please help fix schema on https://github.com/ETDA/etax-schema first. then you can update the schema here with the corrected one. Unless you feel that this should be merged before that. Reopen this PR when the schema is ready.

**Pull request closed without merging**